### DISCUSSION TITLE
Privacy Mode background changer

### DISCUSSION THREAD

**Discussion by RaXorX** [rustdesk/rustdesk#6087] [Feature Request]
It would be nice if we could select the background for the Privacy mode, or even an entirely black screen instead of the purple background with the privacy note.

Same as this but perhaps beeds to be recategorized :-https://github.com/rustdesk/rustdesk/discussions/2919

**Comment by buestad** [comment]
**1 up on this!** 
The purple screen screams! I have my computer in an open office and if I log in using RusDesk in privacy mode it looks like the PC has serious issues or worse, has been hacked! A total black screen with maybe a small text down in a corner showing the ctrl + something to exit privacy mode would be so much better!

**Comment by buestad** [comment]
I dived into the code and did som digging and figured out how to solve this myself!
The code for this is inside a dependecy repos written in c++ which produces the WindowInjection.dll inside the RustDesk main folder. 

_Note: this is for Windows only!_

If anyone wants to use my new discrete privacy screen just head over to my fork on GitHub (https://github.com/buestad/RustDeskTempTopMostWindow/releases/tag/v0.2) and download the v0.2 release and replace your WindowInjection.dll with the one inside the zip-file.

If you want your own custom screen you will need to build this .dll yourself since the picture is actually a byte array inside one of the .cpp source files

**Reply by SoyDiego** [reply]
> I dived into the code and did som digging and figured out how to solve this myself! The code for this is inside a dependecy repos written in c++ which produces the WindowInjection.dll inside the RustDesk main folder.
> 
> _Note: this is for Windows only!_
> 
> If anyone wants to use my new discrete privacy screen just head over to my fork on GitHub (https://github.com/buestad/RustDeskTempTopMostWindow/releases/tag/v0.2) and download the v0.2 release and replace your WindowInjection.dll with the one inside the zip-file.
> 
> If you want your own custom screen you will need to build this .dll yourself since the picture is actually a byte array inside one of the .cpp source files

Hi @buestad, thanks for your work. 
This `dll` continue working in current versions? I need to paste your dll in `C:\Program Files\RustDesk`, and that's it?

Thanks again!

**Reply by buestad** [reply]
I don't know if it still works. I am not using privacy screen anymore. Please test and report. Just replace the WindowInjection.dll in C:\Program Files\RustDesk. If there are no such dll there I guess they have a different way of showing privacy screen now

**Comment by vbmark** [comment]
Hi @buestad 

Thank you for the improved Privacy Mode.  The cursor movement still displays on the remote computer.  Is there a way to disable that?

**Comment by TOMIPortugal** [comment]
@rustdesk We'd also love for this to be implemented. If it is simpler for you, it could be a feature of the Custom Client.

**Comment by lazee486** [comment]
the option to put a static image - ie company logo or message that the pc is being accessed and by who

**Comment by W-i-n-7** [comment]
DEFINETLY!
